src/modules/collections/controllers/collections.controller.ts

```diff
@@ -71,20 +71,15 @@ export class CollectionsController {
 		// Limit access to the objects in the collections
 		const visitorSpaceAccessInfo =
 			await this.ieObjectsService.getVisitorSpaceAccessInfoFromUser(user);
+
 		collections.items.forEach((collection) => {
-			collection.objects = (collection.objects ?? [])
-				.map((object) => {
-					return limitAccessToObjectDetails(object, {
```

Comment:
shouldn't this change also be applied to the other GET route in the collections controller?
https://github.com/viaacode/hetarchief-proxy/blob/feat/ARC-1410/src/modules/collections/controllers/collections.controller.ts#L110